src/backend/api/Fusion.Resources.Api/Authorization/Extensions/RequirementsBuilderExtensions.cs

```diff
@@ -158,52 +158,12 @@ public static IAuthorizationRequirementRule CurrentUserIs(this IAuthorizationReq
             return builder;
         }
 
-        public static IAuthorizationRequirementRule BeContractorInProject(this IAuthorizationRequirementRule builder, ProjectIdentifier project)
-        {
-            builder.AddRule(project, new ContractorInProjectRequirement());
-            return builder;
-        }
-
-        public static IAuthorizationRequirementRule BeContractorInContract(this IAuthorizationRequirementRule builder, Guid orgContractId)
-        {
-            builder.AddRule(new ContractorInContractRequirement(orgContractId));
-            return builder;
-        }
-
-        public static IAuthorizationRequirementRule ProjectAccess(this IAuthorizationRequirementRule builder, ProjectAccess level, ProjectIdentifier project)
-        {
-            builder.AddRule(project, level);
-            return builder;
-        }
         public static IAuthorizationRequirementRule ScopeAccess(this IAuthorizationRequirementRule builder, ScopeAccess level)
         {
             builder.AddRule(level);
             return builder;
         }
 
-        public static IAuthorizationRequirementRule ContractAccess(this IAuthorizationRequirementRule builder, ContractRole role, ProjectIdentifier project, Guid contractOrgId)
-        {
-            var resource = new ContractResource(project, contractOrgId);
-            builder.AddRule(resource, role);
-
-            return builder;
-        }
-
-        public static IAuthorizationRequirementRule DelegatedContractAccess(this IAuthorizationRequirementRule builder, DelegatedContractRole role, ProjectIdentifier project, Guid contractOrgId)
-        {
-            var resource = new ContractResource(project, contractOrgId);
-            builder.AddRule(resource, role);
-
-            return builder;
-        }
-
-        public static IAuthorizationRequirementRule RequestAccess(this IAuthorizationRequirementRule builder, RequestAccess level, QueryPersonnelRequest request)
```

Comment:
Is this related to contracts? (Should it be removed ?)

Reply:
Did not find any refereneces so I just removed it